exps/default/yolov3.py

```diff
@@ -33,11 +33,15 @@ def init_yolo(M):
         return self.model
 
     def get_data_loader(self, batch_size, is_distributed, no_aug=False):
-        from data.datasets.cocodataset import COCODataset
-        from data.datasets.mosaicdetection import MosaicDetection
-        from data.datasets.data_augment import TrainTransform
-        from data.datasets.dataloading import YoloBatchSampler, DataLoader, InfiniteSampler
         import torch.distributed as dist
+        from yolox.data import (
+            COCODataset,
```

Comment:
isort is your good friends, please isort them.

Reply:
ok, thanks